### ISSUE TITLE
In Block Editor, don't add `/` on + click

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#23835]
### Proposed Objective

Accessibility (WCAG)

### Proposed Priority

Priority 1 - Show Stopper

### dotCMS Version

master

### Problem Statement

When user clicks the + button it adds an empty `/` which users will have to remove if no block gets added.

### Steps to Reproduce

1. Go to block editor field
2. Click the +
3. See the `/` being added.

### Acceptance Criteria

Users should be able to use the + button without adding unnecessary text to the field.

**Comment by erickgonzalez** [comment]
`/` is auto-removed if nothing is added.
